src/lib/i18n.json

```diff
@@ -257,12 +257,15 @@
     "project-create-variable-group-cmd-failed": "Create variable group was not successfully executed.",
     "project-create-variable-group-cmd-err-root-invalid": "Project root path is not valid. Provide a valid root path.",
     "project-create-variable-group-cmd-err-variable-group-invalid": "Variable group name is not valid. Provide a valid variable group name.",
-    "project-create-variable-group-cmd-err-bedrock-file-missing": "Bedrock file does not exist. Check that the project directory has been initialized.",
     "project-create-variable-group-cmd-err-file-missing": "The file '{0}' does not exist in project '{1}'.",
     "project-create-variable-group-cmd-err-dependency": "Please run `spk project init` command before running this command to initialize the project.",
 
     "project-init-cmd-failed": "Project init was not successfully executed.",
 
+    "project-append-variable-group-cmd-failed": "Append variable group was not successfully executed.",
+    "project-append-variable-group-cmd-err-dependency": "Please run `spk project init` command before running this command to initialize the project.",
```

Comment:
"project-append-variable-group-cmd-err-bedrock-yaml-missing": "Bedrock.yaml file was not found. Run `spk project init` command to initialize the project.",

Reply:
@dennisseah I'll make the change. However the keys are getting too long, and error messages look hard to parse. I don't see a point of adding "bedrock-yaml-missing" because the message itself includes that information "Bedrock yaml file was not found". To what extent should the key say the message and then the message is repeated in the details.

Reply:
@edaena I do not like long keys too. During the design meeting, it was mentioned that the key has to be description, that's user should be able to guess the error without looking at the messages. Please cross check with @andrebriggs thanks

Reply:
Ok, thanks for the update.